### ISSUE TITLE
[BUG] demo notebook for_analysis.ipynb throws 500 Server Error: Internal Server Error for url:

### ISSUE THREAD

**Issue by DataBoyTX** [graphistry/pygraphistry#572]
**Describe the bug**

User reported this issue, but I recalled they were not using register function from previous conversation.  Some visualizations run without register, but this one fails. 

demo notebook for_analysis.ipynb throws 500 Server Error: Internal Server Error for url: http://... 

**To Reproduce**

Run demo notebook above on private server without register

**Comment by lmeyerov** [comment]
Yeah I'm guessing plot without register, and for complex encodings, maybe missing api=3

**Comment by lmeyerov** [comment]
RE:Automatic login in notebooks,

Looking a bit more, maybe we can shuttle the `graphistry_jwt` cookie in graphistry servers via a notebook extension: https://chatgpt.com/share/550c2c64-e1ae-4009-8fe4-ed67baf86f49

Alternatively, maybe we can make .plot() in an unregistered context pop a login?

(We would also have to strip the GRAPHISTRY_TOKEN etc env vars, which is over due anyways)
